integrations/lib/multiwoven/integrations/protocol/protocol.rb

```diff
@@ -26,6 +26,8 @@ module Types
     LogLevel = Types::String.enum("fatal", "error", "warn", "info", "debug", "trace")
     RequestRateLimitingUnit = Types::String.default("minute").enum("minute", "hour", "day")
     SchemaMode = Types::String.enum("schema", "schemaless")
+    FormatType = Types::String.enum("csv")
```

Comment:
FormatType is a generic name. Format can be anything

Reply:
i will change to File format type